core/api.go

```diff
@@ -141,30 +89,52 @@ type EntryView interface {
 	TTL() time.Duration
 }
 
+// AbstractMapEvent is base for a map event.
+type AbstractMapEvent interface {
+	// Name returns the name of the map for this event.
+	Name() string
+
+	// Member returns the member that fired this event.
+	Member() Member
+
+	// EventType returns the type of entry event.
+	EventType() int32
+
+	// ToString returns a string representation of this event.
+	ToString() string
```

Comment:
Why is this `ToString` ? In Go it should be `String` which is `ToString` in Java.